site-service-client/src/main/java/uk/ac/ox/ndph/mts/siteserviceclient/configuration/WebClientConfigSite.java

```diff
@@ -0,0 +1,100 @@
+package uk.ac.ox.ndph.mts.siteserviceclient.configuration;
+
+import io.netty.channel.ChannelOption;
+import io.netty.handler.timeout.ReadTimeoutHandler;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cloud.client.loadbalancer.LoadBalanced;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.reactive.ClientHttpConnector;
+import org.springframework.http.client.reactive.ReactorClientHttpConnector;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.netty.http.client.HttpClient;
+import reactor.util.retry.Retry;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+@Configuration
+@ConfigurationProperties(prefix = "http-client")
+public class WebClientConfigSite {
+
+    @Value("${http-client.connectTimeOutMs:3000}")
```

Comment:
add todo to add the appropriate config file and add it to the config schema validator